core/state.py

```diff
@@ -1100,6 +1115,13 @@ def SetPwd(self, pwd):
         """Used by builtins."""
         self.pwd = pwd
 
+    def GetSeconds(self):
+        # type: () -> int
+        if self.seconds_state:
+            return self.seconds_state.GetSeconds()
```

Comment:
BTW it feels like this will compute the number of seconds since the FIRST time you invoked `$SECONDS`?

But really it should be since the startup time of bash?

So I think it's OK to eagerly initialize seconds_state in the constructor, which is called in main() I think


```
$ bash -c 'sleep 1; echo $SECONDS'
1
$ bash -c 'sleep 2; echo $SECONDS'
2
```

BTW we could put an "ad hoc" test in `demo/bash-seconds.sh` or something

These are NOT automated, but it's better than nothing.  I don't like to have tests that "sleep" in our automated tests, but in this case it seems hard to test in any other way

The spec test are still useful for the `0` case where you don't sleep, but then we can have another sanity check